beetsplug/playlist.py

```diff
@@ -15,17 +15,16 @@
 import fnmatch
 import os
 import tempfile
-from typing import Any, Optional, Sequence, Tuple
 
 import beets
+from beets.dbcore.query import InQuery
 from beets.util import path_as_posix
 
 
-class PlaylistQuery(beets.dbcore.NamedQuery):
+class PlaylistQuery(InQuery[bytes]):
```

Comment:
See the `bytes` vs `memoryview` discussion.

Reply:
Added back the type mapping to `memoryview` now. Though `bytes` type stays - I found I made a mistake initially assuming it was `str` here.

Reply:
Oh, ok, so this is the same distinction as we discussed regarding the `[Any]SQLiteType` above: The input pattern type is `bytes`, but `col_clause` will turn that into `memoryview` for the SQLite API. It's also related to whether we do this type conversion in the constructor or only in `col_clause` which we discussed elsewhere.